final_prep/annotations.rb

```diff
@@ -3,41 +3,63 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+#defines a method build_a_bear with inputs of name, age, fur, clothes, special_power
```

Comment:
A more technical way to say this would be "defines a method build_a_bear with _parameters_ of name, age, fur, clothes, special_power"